consumer/pdatautil/pdatautil.go

```diff
@@ -19,69 +19,39 @@ package pdatautil
 import (
 	"go.opentelemetry.io/collector/consumer/consumerdata"
 	"go.opentelemetry.io/collector/consumer/pdata"
-	"go.opentelemetry.io/collector/internal/data"
 	"go.opentelemetry.io/collector/translator/internaldata"
 )
 
 // MetricsToMetricsData returns the `[]consumerdata.MetricsData` representation of the `pdata.Metrics`.
 //
 // This is a temporary function that will be removed when the new internal pdata.Metrics will be finalized.
 func MetricsToMetricsData(md pdata.Metrics) []consumerdata.MetricsData {
-	if imd, ok := md.InternalOpaque.(data.MetricData); ok {
-		return internaldata.MetricsToOC(imd)
-	}
-	panic("Unsupported metrics type.")
+	return internaldata.MetricsToOC(md)
 }
 
 // MetricsFromMetricsData returns the `pdata.Metrics` representation of the `[]consumerdata.MetricsData`.
 //
 // This is a temporary function that will be removed when the new internal pdata.Metrics will be finalized.
 func MetricsFromMetricsData(ocmds []consumerdata.MetricsData) pdata.Metrics {
-	return MetricsFromInternalMetrics(internaldata.OCSliceToMetrics(ocmds))
-}
-
-// MetricsToInternalMetrics returns the `data.MetricData` representation of the `pdata.Metrics`.
-//
-// This is a temporary function that will be removed when the new internal pdata.Metrics will be finalized.
-func MetricsToInternalMetrics(md pdata.Metrics) data.MetricData {
-	if ims, ok := md.InternalOpaque.(data.MetricData); ok {
-		return ims
-	}
-	panic("Unsupported metrics type.")
-}
-
-// MetricsFromInternalMetrics returns the `pdata.Metrics` representation of the `data.MetricData`.
-//
-// This is a temporary function that will be removed when the new internal pdata.Metrics will be finalized.
-func MetricsFromInternalMetrics(md data.MetricData) pdata.Metrics {
-	return pdata.Metrics{InternalOpaque: md}
+	return internaldata.OCSliceToMetrics(ocmds)
 }
 
 // CloneMetrics returns a clone of the given `pdata.Metrics`.
 //
 // This is a temporary function that will be removed when the new internal pdata.Metrics will be finalized.
 func CloneMetrics(md pdata.Metrics) pdata.Metrics {
-	if ims, ok := md.InternalOpaque.(data.MetricData); ok {
-		return pdata.Metrics{InternalOpaque: ims.Clone()}
-	}
-	panic("Unsupported metrics type.")
+	return md.Clone()
 }
 
 func MetricCount(md pdata.Metrics) int {
-	if ims, ok := md.InternalOpaque.(data.MetricData); ok {
-		return ims.MetricCount()
-	}
-	panic("Unsupported metrics type.")
+	return md.MetricCount()
 }
 
 func MetricAndDataPointCount(md pdata.Metrics) (int, int) {
-	if ims, ok := md.InternalOpaque.(data.MetricData); ok {
-		return ims.MetricAndDataPointCount()
-	}
-	panic("Unsupported metrics type.")
+	return md.MetricAndDataPointCount()
```

Comment:
Are these functions still needed?

Reply:
Yes because trying to make updating contrib easier, and limit the size of this PR. Will remove soon.